src/calculateRentalCost.js

```diff
@@ -4,7 +4,18 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const price = days * 40;
+  let totalCost = price;
```

Comment:
Consider initializing `totalCost` directly with the discounted price inside the conditional blocks instead of initializing it with `price` and then reassigning it. This can make the code slightly more efficient and clear.